tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts

```diff
@@ -76,6 +76,15 @@ export class FromPixelsProgram implements WebGPUProgram {
     `;
   }
 
+  updateOutputShape(outputShape: number[]) {
```

Comment:
Code looks duplicated with constructor. Call this from the constructor instead of having the code twice?